ouroboros-consensus/docs/ReleaseProcess.md

```diff
@@ -227,3 +227,18 @@ G - fix confusion in the old logic
 B - the release of FOO-1.0.0 (tag: release-FOO-1.0.0)
 A - ...
 ```
+
+# Installing `scriv` 
+
+To manage the workflow described above, we will use the `scriv` tool. If you use
+`nix` then you already will found `scriv` in the `nix develop` shell. Otherwise,
```

Comment:
One still has to use the "old" Nix commands here, as the `flake.nix` is only used by Cicero.
```suggestion
`nix` then you will find `scriv` in the Nix shell. Otherwise,
```